exercises/scale-generator/scale_generator.exs

```diff
@@ -0,0 +1,83 @@
+defmodule ScaleGenerator do
+  @doc """
+  Find the note a given interval (`step`) in `scale` after the `tonic`.
```

Comment:
This reads a little wonky to me. Should it maybe be something like "Find the note for a given interval (`step`) in a `scale` after the `tonic`"?